### PR TITLE
add support for non managed docker networks

### PR DESCRIPTION
**Pull request by hassenius**
This is a initial proposal to address #69 

The proposal works by leveraging the existing labels that are defined in https://github.com/ansible-community/molecule-docker/blob/main/molecule_docker/playbooks/create.yml#L10-L11

If the network already exists the `create` play will not run again, which means that existing networks will not get the `owner: molecule` label applied

Finally the `destroy` play will only destroy networks with the `owner: molecule` labels applied

Not thoroughly tested as I'd like maintainers agreement on approach and feasability

### CODE REVIEW

**Comment by hassenius** [comment]
I have validated it at this point, so it's good to merge if maintainers are happy with approach

**Review by ssbarnea — CHANGES_REQUESTED** [review]
Just by looking at this change I know it will fail linting, mainly because you added tasks files at the same level as molecule playbooks. Please put them inside tasks/ subfolders to make the linter happy.

**Comment by hassenius** [comment]
Suspect it was missing newlines at and of the new files. Tox is passing locally now....
Do you still want the files moved as well?

File: molecule_docker/playbooks/create.yml [hunk 1]
```diff
@@ -93,40 +93,12 @@
...
-        labels: "{{ item.labels }}"
-        name: "{{ item.name }}"
-        scope: "{{ item.scope | default(omit) }}"
-        ssl_version: "{{ item.ssl_version | default(omit) }}"
-        state: "present"
-        timeout: "{{ item.timeout | default(omit) }}"
-        tls: "{{ item.tls | default(omit) }}"
-        tls_hostname: "{{ item.tls_hostname | default(omit) }}"
-        validate_certs: "{{ item.validate_certs | default(omit) }}"
+      include_tasks: network/create_network.yml
```
Comment (ssbarnea) [inline molecule_docker/playbooks/create.yml:96]: please name the folder "tasks"

File: molecule_docker/playbooks/network/delete_network.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+- name: Retrieve network info
+  docker_network_info:
+    name: "{{ item.name }}"
+  register: docker_netname
+
+- name: Delete docker network(s)
+  community.docker.docker_network:
+    name: "{{ item.name }}"
+    state: "absent"
+  when: docker_netname.exists and 'owner' in docker_netname.network.Labels and docker_netname.network.Labels.owner == 'molecule'
```
Comment (ssbarnea) [inline molecule_docker/playbooks/network/delete_network.yml:10]: split this into two lines as AND is implicit between them, make it easier to read the code.

**Review by ssbarnea — APPROVED** [review]

**Pull request merged**